src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -49,6 +49,7 @@
  * <li>
  * Property {@code max} - Specify the maximum threshold allowed.
  * Default value is {@code 20}.
+ * Type - {@code integer}
```

Comment:
Type should be `Integer`  to be exact and match java type

Reply:
Done